addons/kube-ovn/enable

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+VERSION=`cat ../../addons.yaml | grep kube-ovn -A3 | grep version | cut -d ":" -f 2 | tr -d '"' | xargs`
```

Comment:
@ktsakalozos is this the correct directory pathing? I had an issue once were it seemed to be only a single directory below.